### DISCUSSION TITLE
Is it possible to use this library as a modal screen for Bottom Tab Navigator (by React Navigation) ?

### DISCUSSION THREAD

**Discussion by daviz00** [gorhom/react-native-bottom-sheet#564] [Q&A]
I am currently facing an issue in implementing this feature. 
Any help will be highly appreciated.

**Comment by RemyMachado** [comment]
Probably a bit late, but I made an example [here](https://github.com/gorhom/react-native-bottom-sheet/issues/249#issuecomment-906535858)
